### PR TITLE
[DQL2-13] CKAN Harvest - Install and configure

### PR DESCRIPTION
**Pull request by salsa-nathan**
- Added `ckanext-harvest` to `requirements-dev.txt`
- Added HarvestExtension settings to `templates/Salsa-CKAN-Extensions.cfn.yml`
- Added `CKANExtHarvest` to `vars/shared-OpenData.var.yml`

cc: @duttonw & @ThrawnCA

### CODE REVIEW

File: templates/Salsa-CKAN-Extensions.cfn.yml [hunk 1]
```diff
@@ -99,6 +99,14 @@ Parameters:
     Description: Branch/tag name in repository to checkout.
     Default: master
     Type: String
+  HarvestExtensionSource:
```
Comment (duttonw) [inline templates/Salsa-CKAN-Extensions.cfn.yml:102]: hmm, i need to double check, i think this is now not in use.

File: requirements-dev.txt [hunk 2]
```diff
@@ -14,3 +14,4 @@ splinter>=0.13.0
 -e git+https://github.com/ckan/ckantoolkit@release-0.0.4#egg=ckantoolkit
 -e git+https://github.com/ckan/ckanapi@ckanapi-4.3#egg=ckanapi
 -e git+https://github.com/ckan/ckanext-scheming@release-1.2.0#egg=ckanext-scheming
+-e git+https://github.com/ckan/ckanext-harvest.git@v1.2.0#egg=ckanext-harvest
```
Comment (duttonw) [inline requirements-dev.txt:17]: would be awesome if this file was built dynamicly off  the vars opendata variables.

File: templates/Salsa-CKAN-Extensions.cfn.yml (hunk 1 above)
Comment (duttonw) [reply templates/Salsa-CKAN-Extensions.cfn.yml:102]: @ThrawnCA  is this now not in use?

Comment (ThrawnCA) [reply templates/Salsa-CKAN-Extensions.cfn.yml:102]: Correct, this file is no longer used. The only one that needs to be updated is `shared-OpenData.var.yml`

Comment (salsa-nathan) [reply templates/Salsa-CKAN-Extensions.cfn.yml:102]: should this file just be deleted in the PR?

**Pull request merged**